### ISSUE TITLE
It seems like the add() function is not accessible? 

### ISSUE THREAD

**Issue by monbro** [LeeXun/word2vector#2]
Using "word2vector": "^2.0.0"
I tried the following code:
```
var w2v = require( 'word2vector' );
var modelFile = 'GoogleNews-vectors-negative300.bin';
w2v.load( modelFile );
// console.log(w2v.getSimilarWords("yoga")); // works properly

var a = w2v.add("health", "yoga");
console.log(a);
```
and got the following error:
```
./node_modules/word2vector/lib/calcs.js:57
    if(!this.model.hasOwnProperty(w1)) return false; // error log in getVector;
                   ^

TypeError: Cannot read property 'hasOwnProperty' of undefined
    at Object.add (nlp-feature/node_modules/word2vector/lib/calcs.js:57:20)
    at Object.<anonymous> (nlp-feature/word2vector.js:15:13)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

It seems like the add() function is not accessible?

**Comment by monbro** [comment]
Using
````
var o = w2v.getVector('yoga');
var p = w2v.getVector('health');
var a = w2v.add(o, p);
var b = w2v.getNeighbors(a, {N:10});
console.log(b);
````
works now. Probably you want to correct this in your documentation / examples.

**Comment by LeeXun** [comment]
@monbro Thank you. I will check this later.
Btw, Glad this project can help you guys out.
I am planning to refactor and review the project this week.

**Comment by LeeXun** [comment]
@monbro Thank you. I will check this later.

**Comment by LeeXun** [comment]
Issue fixed at version 2.0.1.
That's an old bug that I forget to update.
Thanks again : )

**Comment by LeeXun** [comment]
Issue fixed at version 2.0.1.
That's an old bug that I forget to update.
Thanks again : )
